tcf_website/management/commands/load_grades.py

```diff
@@ -0,0 +1,398 @@
+
+        self.load_dict_into_models()
+
+    def load_row_into_dict(self, row):
+        try:
+            # parsing fields of excel
+
+            first_name = row['Instructor First Name']
+            middle_name = row['Instructor Middle Name']
+            last_name = row['Instructor Last Name']
+            email = row['Instructor Email']
+            subdepartment = row['Subject']
+            number = re.sub('[^0-9]', '', str(row['Course Number']))
+            section_number = row['Section Number']  # not used
+            title = row['Title']
+            gpa = float(row['Course GPA'])
+            a_plus = int(row['A+'])
+            a = int(row['A'])
+            a_minus = int(row['A-'])
+            b_plus = int(row['B+'])
+            b = int(row['B'])
+            b_minus = int(row['B-'])
+            c_plus = int(row['C+'])
+            c = int(row['C'])
+            c_minus = int(row['C-'])
+            d_plus = int(row['D+'])
+            d = int(row['D'])
+            d_minus = int(row['D-'])
+            f = int(row['F'])
+            ot = int(row['OT'])
+            drop = int(row['DR'])
+            withdraw = int(row['W'])
+            # credit
+            # general_credit
+            # no_credit
+            total_enrolled = int(row['Total'])
+
+            # identifiers are tuple keys to dictionaries
+            course_identifier = (subdepartment, number, title)
+            course_instructor_identifier = (
+                subdepartment, number, first_name, middle_name, last_name, email)
+
+            # value of dictionaries (incremented onto value if key already
+            # exists)
+            this_semesters_grades = [
+                a_plus,
+                a,
+                a_minus,
+                b_plus,
+                b,
+                b_minus,
+                c_plus,
+                c,
+                c_minus,
+                d_plus,
+                d,
+                d_minus,
+                f,
+                ot,
+                drop,
+                withdraw]
+
+            # load this semester into course dictionary
+            if course_identifier not in course_grades:
+                course_grades[course_identifier] = this_semesters_grades
+            else:
+                for i in range(len(course_grades[course_identifier])):
+                    course_grades[course_identifier][i] += this_semesters_grades[i]
+
+            # load this semester into course instructor dictionary
+            if course_instructor_identifier not in course_instructor_grades:
+                course_instructor_grades[course_instructor_identifier] = this_semesters_grades
+            else:
+                for i in range(
+                        len(course_instructor_grades[course_instructor_identifier])):
+                    course_instructor_grades[course_instructor_identifier][i] += this_semesters_grades[i]
+
+        except TypeError as e:
+            print(row)
+            print(e)
```

Comment:
Same thing about verbose mode

Reply:
Done in 2b434da273891c45744aeecfd86df860dbb10112